certs/readme.md

```diff
@@ -0,0 +1,22 @@
+## Certificates - Important to know
+
+For sample purposes, C SDK samples leverage certificates on the `certs.c` file. When creating solutions based on the C SDK you should consider :
+
+1. For production environments, the IoT Hub certificate must be validated during the TLS handshake.
+2. We use the Baltimore root CA to validate IoT Hub server certificate (for the regions that use this certificate)
+3. For other regions (and private cloud environments) the appropriate root CA shall be used
+
+## Certificate update strategy
+
+As shared on [this blog post](https://techcommunity.microsoft.com/t5/internet-of-things/azure-iot-tls-changes-are-coming-and-why-you-should-care/ba-p/1658456), Microsoft began phasing out her older PKI provided by Baltimore Cybertrust Root this August in favor of new PKI providers. For Azure IoT, this means that Azure IoT services will serve TLS certificates that will be rooted to the following:
```

Comment:
Microsoft began phasing out **its**

Reply:
I'm a bit confused by the statement: the referenced blog is saying that no action is required and that some of the intermediate certificate authorities are changing between Aug and Nov 2020.

Reply:
we removed the redundant information from this file and added a link to the official source of information: the blog post.